docs/data/charts/zoom-and-pan/ZoomControlled.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import { LineChartPro } from '@mui/x-charts-pro/LineChartPro';
 import { ZoomData } from '@mui/x-charts-pro/context';
-import { Button } from '@mui/base';
+import Button from '@mui/material/Button';
```

Comment:
@mui/xcharts WDYT about this change? 🤔 
I've tried using `<button />` initially, but that calls for explicit `type="button"`, which increases the line count in this demo, and removes the preview due to this. 🙈 
I see 2 options:
- Use `Button` from `@mui/material`
- Create a local `Button` component defaulting the `type` to `button`

I've gone with the first approach for simplicity and uniformity among the demos.
What's your take?

Reply:
It is looking good. Thanks for taking care of it 💪

Reply:
Using mateiral button is good for me. Most of our demo are already using components from material :+1: